modules/dnn/src/tflite/tflite_importer.cpp

```diff
@@ -939,6 +939,8 @@ void TFLiteImporter::parseActivation(const Operator& op, const std::string& opco
                 y = std::min(std::max(x, 0.f), 6.f);
             else if (opcode == "LOGISTIC")
                 y = 1.0f / (1.0f + std::exp(-x));
+            else if (opcode == "HARD_SWISH")
+                y = x * std::min(std::max(x + 3.f, 0.f), 6.f) * 0.166666667;
```

Comment:
Formula is different from https://github.com/opencv/opencv/blob/76548e29bd62f299386cb919824b34ce4baf73cb/modules/dnn/src/layers/elementwise_layers.cpp#L1900